packages/lit-html/src/lit-html.ts

```diff
@@ -1496,6 +1496,26 @@ export type {EventPart};
 class EventPart extends AttributePart {
   readonly type = EVENT_PART;
 
+  constructor(
+    element: HTMLElement,
+    name: string,
+    strings: ReadonlyArray<string>,
+    parent: Disconnectable,
+    options: RenderOptions | undefined
+  ) {
+    super(element, name, strings, parent, options);
+
+    if (
+      DEV_MODE &&
+      (strings.length !== 2 || strings[0] !== '' || strings[1] !== '')
+    ) {
+      throw new Error(
+        'An event listener in a template must have exactly one expression ' +
```

Comment:
Use the name in the error for a little more hint to the developer as to where the problem is.

Reply:
Updated, here's an example:
``A `<div>` has a `@click=...` listener with invalid content. Event listeners in templates must have exactly one expression and no surrounding text.``
WDYT?

Reply:
+1 sounds great